react/src/Stores/BaseStore.js

```diff
@@ -286,7 +288,19 @@ class EventStore extends BaseStore {
         });
 
         socket.on('OnWarning', (data) => {
-            ErrorActions.setToastError(data.Msg);
+            if(data.Code === 2) {
```

Comment:
Maybe define `2` as a constant? You might also want to add a link to the docs to explain why `2`.

Reply:
see d5c611e